src/main/java/bio/terra/service/resourcemanagement/google/GoogleResourceConfiguration.java

```diff
@@ -96,33 +55,11 @@ public void setAllowReuseExistingBuckets(boolean allowReuseExistingBuckets) {
     this.allowReuseExistingBuckets = allowReuseExistingBuckets;
   }
 
-  public String getDefaultFirestoreLocation() {
-    return defaultFirestoreLocation;
-  }
-
-  public void setDefaultFirestoreLocation(String defaultFirestoreLocation) {
-    this.defaultFirestoreLocation = defaultFirestoreLocation;
-  }
-
```

Comment:
Why don't we need this anymore?  This seems more related to region work than project creation.

Reply:
I just noticed it was unused so I deleted it. Since we create a firestore for each dataset/snapshot, there's no longer a default firestore region, since it's always inherited from the dataset/snapshot region